video_xblock/backends/youtube.py

```diff
@@ -3,9 +3,9 @@
 YouTube Video player plugin.
 """
 
-import HTMLParser
+from html.parser import HTMLParser
 import json
-import httplib
+import http.client as http_client
```

Comment:
Please rewrite imports in alphabetical order.

Reply:
done